tests/components/panel_custom/test_init.py

```diff
@@ -151,16 +151,44 @@ async def test_module_webcomponent(hass):
     assert panel.sidebar_title == "Sidebar Title"
 
 
+async def test_latest_and_es5_build(hass):
+    """Test specifying an es5 and latest build."""
+    config = {
+        "panel_custom": {
+            "name": "todo-mvc",
+            "js_url": "/local/es5.js",
+            "module_url": "/local/latest.js",
+            "url_path": "nice_url",
+        }
+    }
+
+    assert await setup.async_setup_component(hass, "panel_custom", config)
+
+    panels = hass.data.get(frontend.DATA_PANELS, [])
```

Comment:
`panels` should be a dict.
```suggestion
    panels = hass.data.get(frontend.DATA_PANELS, {})
```